clients/mediaconvert.go

```diff
@@ -121,8 +125,60 @@ func (mc *MediaConvert) Transcode(ctx context.Context, args TranscodeJobArgs) er
 		srcInputFile = mc.s3TransferBucket.JoinPath(mcInputRelPath)
 	}
 
+	// temporarily probe input mp4 here...
```

Comment:
Note: we know is pretty ugg - using a follow-up PR to move all of this parsing logic into the `video/probe.go` code to abstract all of it. Calling the probe functions will just return a pre-filled `InputVideo` struct after the refactor and will make it easier to integrate in the Mist pipeline. 
cc @thomshutt @victorges